### ISSUE TITLE
oxf.xforms.format.input.date/time can't depend on language

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#1152]
When switching between Italian and German for example (client use case), one might want to change the format.

Currently:
- used by `xforms.js` to format values into the field
- used by `xforms.js` to know whether day or month is first when parsing
- initialized [only once](https://github.com/orbeon/orbeon-forms/blob/92de738da713e00a18d113b4d6fe21d0f6132e17/src/resources-packaged/ops/javascript/xforms.js#L915)

**Comment by ebruchez** [comment]
Closing as duplicate of #2991.
